tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -103,16 +103,6 @@ def rhsso_groups_teardown(default_sso_host):
         default_sso_host.delete_rhsso_group(group_name)
 
 
-@pytest.fixture
-def multigroup_setting_cleanup(default_ipa_host):
-    """Adding and removing the user to/from ipa group"""
-    sat_users = settings.ipa.groups
-    idm_users = settings.ipa.group_users
-    default_ipa_host.add_user_to_usergroup(idm_users[1], sat_users[0])
-    yield
-    default_ipa_host.remove_user_from_usergroup(idm_users[1], sat_users[0])
```

Comment:
Isn't it required now to add `satuser_01` in the `satadmin` group? If not, would you mind describing why?

Reply:
User `satadmin` is in `satusers` group, that is stable and doesn't need to be done everytime.
Although you made me realise that one of the users in tests should actually be `satadmin` for the test to cover what it should cover, so thanks for this question!
Also, I will rerun PRT.